catalog/security/catalog-security-metacardattributeplugin/src/main/java/org/codice/ddf/catalog/security/policy/metacard/MetacardAttributeSecurityPolicyPlugin.java

```diff
@@ -36,24 +40,42 @@
  * Plugin that parses Metacard attributes for security policy information
  */
 public class MetacardAttributeSecurityPolicyPlugin implements PolicyPlugin {
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            MetacardAttributeSecurityPolicyPlugin.class);
+
+    private List<String> metacardAttributes = new ArrayList<>();
 
-    List<String> metacardAttributes = new ArrayList<>();
+    private Map<String, String> mcAttribsLookup = new HashMap<>();
```

Comment:
Any concerns on threading here with this map? Assuming the answer is no, but just checking.